handlers.js

```diff
@@ -79,6 +79,22 @@ async function rank(c, player, num) {
     }
 }
 
+function reply_wait(f)
```

Comment:
I like the idea of this function - I do think it would be nice to be able to edit the message instead though. It looks like I've definitely over-abstracted some of this stuff and it doesn't fit so nicely with messages like this. If you can think of a nice way to add that functionality without changing too much that would be great, but if not its no big deal - this is fine!